website/.remarkrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  plugins: [
+    [
+      'remark-lint-no-dead-urls',
+      {
+        // This feature is currently only supported in ^2.0.0 versions of check-links, which is strictly
+        // ESM only. For now we have to exclude checking urls that redirect using 302/303s.
+        gotOptions: {
+          methodRewriting: true,
+        },
+        skipUrlPatterns: [
+          // Returns 302/303 responses
+          /developer\.android\.com/,
+          // Has an CAPTCHA
+          /apkfiles\.com/,
+          /localhost/,
```

Comment:
```suggestion
          /localhost/,
          /github\.com/
```

Large part of false-positive warnings in the new check originate from GitHub URLs, so probably we need to skip those links too.

Reply:
Hmm, for some reason the problematic GitHub links include `'` character at the end. 🤔

Reply:
Taking a closer look, they're all HTTP 302 redirects (there's a dependency that needs a major update, but it's non-trivial: cjs → mjs).  I really wanted to  avoid having to fork `remark-lint-no-dead-urls`, but let me see if I can do it quickly.

Reply:
Forked it in the end.